vault/data_source_aws_access_credentials.go

```diff
@@ -115,6 +115,11 @@ func awsAccessCredentialsDataSource() *schema.Resource {
 				Computed:    true,
 				Description: "True if the duration of this lease can be extended through renewal.",
 			},
+			"ttl": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Human-friendly description of the mount for the backend.",
```

Comment:
the description seems a bit cryptic, or is it just me?

Reply:
Whoops, that's a bad copy-paste. I updated it in ae5e7aa8 , thanks for catching that!